src/pages/workspaces/migration/BillingProjectParent.test.ts

```diff
@@ -53,6 +97,8 @@ describe('BillingProjectParent', () => {
     );
     expect(await axe(container)).toHaveNoViolations();
     await user.click(screen.getByText('Migrate all workspaces'));
+    // Confirmation dialog
+    await user.click(screen.getByText('Migrate All'));
```

Comment:
Didn't check the dialog text in this case because it's the same as the previous, where we cancelled.